package-whitelist.json

```diff
@@ -22,6 +22,11 @@
   "vendor/tribe-select2/*.js",
   "vendor/tribe-select2/*.gif",
   "vendor/tribe-select2/*.png",
+  "vendor/autoload.php",
+  "vendor/autoload_52.php",
+  "vendor/composer/*",
+  "vendor/lucatume/di52/src/**",
+  "vendor/a5hleyrich/wp-background-processing/**",
```

Comment:
We should probably only permit the files that are necessary for packaging `*.php` & `license.txt` for wp-background-processing. We can probably exclude .gitignore, composer.json, and the readme

Reply:
yep, definitely

Reply:
Ok added in https://github.com/moderntribe/tribe-common/pull/662/commits/a86107eeeab39fdccb536cee13275272286e7cb7